packages/paste-core/components/callout/src/Callout.tsx

```diff
@@ -43,33 +45,48 @@ export interface CalloutProps extends HTMLPasteProps<"div"> {
    * @memberof CalloutProps
    */
   marginY?: BoxStyleProps["marginY"];
+  /**
+   * Function to run on dismiss of the Callout. Adds a close button.
+   *
+   * @default null
+   * @memberof CalloutProps
+   */
+  onDismiss?: () => void;
+  /**
+   * Title for dismiss label. Only necessary when using onDismiss.
+   *
+   * @default 'Dismiss callout'
+   * @memberof CalloutProps
+   * @type {string}
+   */
+  i18nDismissLabel?: string;
 }
 
 const variantStyles: Record<CalloutVariants, BoxStyleProps> = {
   success: {
     backgroundColor: "colorBackgroundSuccessWeakest",
     color: "colorTextSuccess",
-    borderColor: "colorBorderSuccessWeaker",
+    borderColor: "colorBorderSuccessWeak",
   },
   error: {
     backgroundColor: "colorBackgroundErrorWeakest",
     color: "colorTextError",
-    borderColor: "colorBorderErrorWeaker",
+    borderColor: "colorBorderErrorWeak",
   },
   warning: {
     backgroundColor: "colorBackgroundWarningWeakest",
     color: "colorTextWarningStrong",
-    borderColor: "colorBorderWarningWeaker",
+    borderColor: "colorBorderWarningWeak",
   },
   new: {
     backgroundColor: "colorBackgroundNewWeakest",
     color: "colorTextNew",
-    borderColor: "colorBorderNewWeaker",
+    borderColor: "colorBorderNewWeak",
   },
   neutral: {
     backgroundColor: "colorBackgroundNeutralWeakest",
     color: "colorTextNeutral",
-    borderColor: "colorBorderNeutralWeaker",
+    borderColor: "colorBorderNeutralWeak",
```

Comment:
todo: All Callout backgrounds (except the error variant) should now be colorBackgroundWeak